Tasks/PublishTestResults/Tests/TestSetup.ts

```diff
@@ -0,0 +1,67 @@
+import  * as ma from 'vsts-task-lib/mock-answer';
+import * as tmrm from 'vsts-task-lib/mock-run';
+import * as tmt from 'vsts-task-lib/mock-task';
+import * as constants from './Constants';
+import * as path from 'path';
+
+// Get the task path
+const taskPath = path.join(__dirname, '..', 'publishtestresults.js');
+const tr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+// Set inputs
+tr.setInput('testRunner', process.env[constants.testRunner]);
+tr.setInput('testResultsFiles', process.env[constants.testResultsFiles]);
+tr.setInput('mergeTestResults', process.env[constants.mergeTestResults]);
+tr.setInput('platform', process.env[constants.platform]);
+tr.setInput('configuration', process.env[constants.configuration]);
+tr.setInput('testRunTitle', process.env[constants.testRunTitle]);
+tr.setInput('publishRunAttachments', process.env[constants.publishRunAttachments]);
+tr.setInput('searchFolder', process.env[constants.searchFolder]);
+
+const PublishExeToolPath = path.join(__dirname, '../modules', 'TestResultsPublisher.exe');
+const PublishExeArgs = '@' + path.join(__dirname, '..', 'tempResponseFile.txt');
```

Comment:
why hardcoding path? good to generate temp file with guid [in agent_temp] location and pass it to task. this file will be deleted automatically by agent.